### PR TITLE
TileLayer loading attribute example and doc

### PR DESCRIPTION
**Pull request by fitoprincipe**

### CODE REVIEW

**Comment by martinRenou** [comment]
Thanks a lot! Is it ready to be merged?

**Comment by fitoprincipe** [comment]
If you agree with the name of the example notebook and with the comment on docs, yes! =)

**Comment by fitoprincipe** [comment]
I don't understand why it takes the first two commits that were already merged :thinking:

File: docs/source/api_reference/tile_layer.rst [hunk 1]
```diff
@@ -61,4 +61,5 @@ opacity            1.0
 visible            True
 no_wrap            False
 show_loading       False
+loading            False (dynamically updated)
```
Comment (martinRenou) [inline docs/source/api_reference/tile_layer.rst]: This makes me think that the `loading` attribute should be specified as read-only. Something like:

```python
loading = Bool(False, read_only=True).tag(sync=True)
```

So that users would have an error when they try to assign to it. Could be in a separate PR though, as you wish.

**Comment by martinRenou** [comment]
> I don't understand why it takes the first two commits that were already merged

That's fine, I can squash commits when merging :)

I would suggest creating branches next time instead of committing to the master branch. That would prevent this :)

File: docs/source/api_reference/tile_layer.rst (hunk 1 above)
Comment (fitoprincipe) [reply docs/source/api_reference/tile_layer.rst]: Yes, I though of that. I'll do it right now and see if doesn't break anything. If it doesn't, then I'll commit again.

Comment (fitoprincipe) [reply docs/source/api_reference/tile_layer.rst]: `Cannot find module "leaflet.awesome-markers"` in JS console... :thinking:

Comment (martinRenou) [reply docs/source/api_reference/tile_layer.rst]: Oh yes, it's a new dependency that has been added. You need to run `npm install` to install it.

**Comment by fitoprincipe** [comment]
> I would suggest creating branches next time instead of committing to the master branch. That would prevent this :)

OK!

File: npm-debug.log [hunk 2]
```diff
@@ -0,0 +1,95 @@
+0 info it worked if it ends with ok
```
Comment (martinRenou) [inline npm-debug.log]: You might have committed this file by mistake?

Comment (fitoprincipe) [reply npm-debug.log]: that is weird, did not appear when I did `git diff`, besides, I did not create it manually, so it is created automatically.. but it is not ignored in `.gitignore`..

Comment (fitoprincipe) [reply npm-debug.log]: can you help me on this? I did:

``` bash
git rm --cached npm-debug.log
git add -A
git commit -m 'remove file commited by mistake'
git push
```
But did not worked as expected :thinking:

Comment (martinRenou) [reply npm-debug.log]: Which part did not work?

Comment (fitoprincipe) [reply npm-debug.log]: I don't see my commit here :thinking:... I pushed it

Comment (martinRenou) [reply npm-debug.log]: Are you sure `git push` did not give an error?

**Comment by fitoprincipe** [comment]
how does it look now? By the way, why the npm log file was created on that folder?

**Comment by martinRenou** [comment]
I guess because you ran 'npm run build' from the root directory at some point..

Thanks for the PR!

**Comment by fitoprincipe** [comment]
May be I run `npm run watch` accidentally in that folder. OK. Thanks! =)
